internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -155,7 +155,7 @@ func (e *EnvStackConfig) ToEnv(stack *cloudformation.Stack) (*archer.Environment
 
 	return &archer.Environment{
```

Comment:
Are we going to tackle this later, moving to `config.Environment`? I think that's fine, just wanted to make sure.